### DISCUSSION TITLE
New antispam system being tested

### DISCUSSION THREAD

**Discussion by Kurdakov** [ThioJoe/YT-Spammer-Purge#897] [General]
On twitter found this message:

https://twitter.com/JayzTwoCents/status/1553784121075257344

> So I have been in the closed testing group for anti spam filters on YouTube and it's been a week since I have received any messages from people telling me they got a reply from the spam bots. It appears that for now, it's greatly reduced. Until the bots adapt that is.

seems YouTube finally starts to address issue seriously and they have some success. Good news.

**Comment by Firecul** [comment]
It *is* much better, normally I'd expect a lot more spam from 3000 comments 
![image](https://user-images.githubusercontent.com/34950371/182242404-2dd0cb85-aa88-438a-b644-43040ac713f3.png)

**Comment by Kurdakov** [comment]
finally it starts to be deployed to youtubers:

https://twitter.com/SnazzyQ/status/1562688747627372546
